### PR TITLE
[MODULAR] TG Aesthetic Revert

### PR DESCRIPTION
**Pull request by W4rd3nn**
## About The Pull Request
A lot of the sprites we use are from #1245 

These are good, but some are starting to show their age or have other issues. TG has had a couple great resprites, some a product of the wallening, others just because. Either way, this PR reverts the old aesthetics change for ~~crates~~, air alarms, and fire alarms. This will bring us more in line with the wallening should it ever actually happen.

## How This Contributes To The Skyrat Roleplay Experience
Good sprites make thing more aesthetic heehaw

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
~~New crates (some random other stuff i spawned too)~~ REMEMBER WHAT THEY TOOK FROM YOU
dreamseeker_2023-10-02_15-29-06

~~Wrapped Crates~~
image

Air alarm in da dark
dreamseeker_2023-10-02_16-45-42

Orange alarm
dreamseeker_2023-10-02_16-59-02

Fire alarm and air alarm
dreamseeker_2023-10-02_17-16-14

Violet alarm
dreamseeker_2023-10-02_17-18-54

</details>

## Changelog

:cl: Motho
image: Sector 13's station air alarms and fire alarms have been updated to be more in line with other frontier station models.
/:cl:

### CODE REVIEW

**Comment by W4rd3nn** [comment]
not actually a full revert so we shouldnt have a revert label on this....

**Comment by RatFromTheJungle** [comment]
everything but the crates is a flat upgrade, but those crates are just something else

File: modular_skyrat/modules/aesthetics/crates/code/crates.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/aesthetics/crates/code/crates.dm:1]: I'm sorry, but I just don't like the new crates, they don't seem to match the style of the rest of the stuff in-game. The black outlines on the side just don't feel like they fit well here.

**Review by GoldenAlpharex — DISMISSED** [review]
I'm not a fan of the /tg/ alarms, but I'm not going to oppose to it super hard.

I don't want /tg/'s crates, however.

File: modular_skyrat/modules/aesthetics/crates/code/crates.dm
Comment (W4rd3nn) [reply modular_skyrat/modules/aesthetics/crates/code/crates.dm:1]: boowomp...

**Comment by HoodMilk** [comment]
Tg crates look so bad but the alarm sprites are nice.

**Comment by Arctaisia** [comment]
we feel we prefer SR's current air alarms a lot more, though the fire alarm is pretty good. 

as much as we don't hate the crates too, they don't fit with SR's style as good as current ones, (sorta) same for air alarms, but it isn't a huge difference

fire alarms are good tho

**Review by lessthnthree — APPROVED** [review]

**Pull request merged**